src/Settings.cpp

```diff
@@ -367,15 +380,18 @@ void Settings::parseInput(int argc, char** argv)
             break;
         case 'l':  // loopback
             //-------------------------------------------------------
-            mLoopBack = true;
+            mGuiIgnoresArgemunts = true;
```

Comment:
Typo: mGuiIgnoresArguments

Reply:
I was wondering how it even managed to compile with such a typo, then realised I must have made it the first time and autocompleted every other time.